setup.py

```diff
@@ -59,8 +59,6 @@ def get_readme_content():
             # To support CronJob api versions 'batch/v1beta1' & 'batch/v1'
             'kubernetes>=21.7.0,<25.0.0',
             'tabulate<1',
-            # compose v2 has broken container naming
-            'docker-compose<2',
```

Comment:
`'docker-compose<2'`, should not be `'docker-compose,'` ?

Reply:
Docker-compose v2 isn't a python package anymore, I think it's go? Or it's the part of the docker package? I need to refresh my memory on that...

Reply:
https://docs.docker.com/compose/migrate/

Reply:
`compose` is now a subcommand of `docker` itself, and consequently is a Go package that comes with every new Docker installation. There isn't _technically_ `docker-compose` v2, there's `docker-compose` which is a Python package, and `compose` which is a feature of `docker`.